docs/src/rules/func-style.md

```diff
@@ -44,7 +44,7 @@ var doSomething = function() {
 };
 ```
 
-In this case, `doSomething()` is undefined at the time of invocation and so causes a runtime error.
+In this case, `doSomething()` is `undefined` at the time of invocation and so causes a runtime error.
```

Comment:
```suggestion
In this case, `doSomething` is `undefined` at the time of invocation and so causes a runtime error.
```

I think this would be more correct.